README.md

```diff
@@ -94,13 +94,16 @@ The community has also started developing third-party providers for Rustls:
 cryptography.
 * [`rustls-rustcrypto`] - an experimental provider that uses the crypto primitives
 from [`RustCrypto`] for cryptography.
+* [`rustls-post-quantum`]: an experimental provider that adds support for post-quantum
+key exchange to the default aws-lc-rs provider.
 
 [`rustls-mbedtls-provider`]: https://github.com/fortanix/rustls-mbedtls-provider
 [`mbedtls`]: https://github.com/Mbed-TLS/mbedtls
 [`boring-rustls-provider`]: https://github.com/janrueth/boring-rustls-provider
 [`boringssl`]: https://github.com/google/boringssl
 [`rustls-rustcrypto`]: https://github.com/RustCrypto/rustls-rustcrypto
 [`RustCrypto`]: https://github.com/RustCrypto
+[`rustls-post-quantum`]: https://crates.io/crates/rustls-post-quantum
```

Comment:
If `rustls-post-quantum` is going to be published as a separate crate should its metadata have `publish = false`? (I haven't found an upstream docs page for what this option does yet)

https://github.com/rustls/rustls/blob/62e154cb997a81a1425ec99cf1b8da69acf49dd8/rustls-post-quantum/Cargo.toml#L5

Reply:
Aha, [here are the docs](https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field)

I think that would have to be removed.

Reply:
I will follow up with another PR that removes `publish = false`, and sets a version (rather than path) dependency on rustls.

Reply:
SGTM